app/scripts/main.js

```diff
@@ -67,7 +67,7 @@ var iViz = (function(_, $, cbio, QueryByGeneUtil, QueryByGeneTextArea) {
         attrData.group_type = 'patient';
         if (chartsCount < 20 && patientChartsCount < 10) {
           if (attrData.show) {
-            attrData.group_id = vm_.groupCount;
+            attrData.group_id = vm_.groupCount.toString();
```

Comment:
I notice group_id is not always string or number throughout the project which cases some issues